Bio/Seq.py

```diff
@@ -1525,6 +1571,8 @@ def translate(
         >>> print(my_protein)
         XXX
 
+        Note even if your sequence makes sense as codons (e.g. AAA), it will
+        be translated as an unknown protein sequence of X.
```

Comment:
Do you think ``UnknownSeq(9, character="U").translate()`` should give ``UnknownSeq(3, character='X')`` (current behaviour) or ``UnknownSeq(3, character='F')`` based on the codon ``UUU``?

Reply:
`UnknownSeq(3, character='F')`. Otherwise `UnknownSeq(9, character="U") == 'UUUUUUUUU' == Seq('UUUUUUUUU')` but `UnknownSeq(9, character="U").translate() != translate('UUUUUUUUU') == Seq('UUUUUUUUU').translate()`

Reply:
Done.